collections/search/index.php

```diff
@@ -567,31 +569,15 @@
 		ul.outerWidth(this.element.outerWidth());
 	}
 	const collectionSource = <?php echo isset($collectionSource) ? json_encode($collectionSource) : 'null'; ?>;
+	const collIdsFromUrl = <?php echo isset($collIdsFromUrl) ? json_encode($collIdsFromUrl) : 'null'; ?>;
```

Comment:
I think this is safe current based on its use however the contents of this can be unsafe so just fyi I guess